doc/guide/working_with_elements.rst

```diff
@@ -135,6 +135,49 @@ Accessing custom elements is much like accessing inline ones:
         Page factory takes care of creating custom elements and their class names
         follow the same rules as Page class names.
 
+Elements can be nested, so similar to how elements can be retrieved from a `Page`, elements can also be
+retrieved from an element:
+
+
+    .. code-block:: php
+
+        <?php
+
+        namespace Page\Element;
+
+        use SensioLabs\Behat\PageObjectExtension\PageObject\Element;
+        use SensioLabs\Behat\PageObjectExtension\PageObject\InlineElement;
+        use SensioLabs\Behat\PageObjectExtension\PageObject\Page;
+
+        class Header extends Element
+        {
+            /**
+             * @var array|string $selector
+             */
+            protected $selector = '.header';
+
+            protected $elements = [
+                'Logo' => '#logo',
+            ];
+
+            /**
+             * @return SearchForm
+             */
+            public function searchForm()
+            {
+                return $this->getElement(SearchForm::class);
+                // or (for PHP < 5.5.0) return $this->getElement('Page\\SearchForm')->search($keywords);
```

Comment:
It might be time to stop documenting PHP < 7.0

Reply:
We don't support it anymore so I strongly agree

Reply:
Removed!